Makefile

```diff
@@ -229,3 +229,23 @@ check-tidy:
 		mv go.sum.tidy-check go.sum; \
 		exit 1 \
 	)
+
+#
+# Release
+#
+
+.PHONY: new-version
+new-version: check-standard-version
+	npx standard-version
+
+.PHONY: next-version
+next-version: check-standard-version
+	npx standard-version --dry-run
+
+.PHONY: check-standard-version
+check-standard-version:
+	$(if $(shell which npx),,\
+		$(error No npx found in PATH, please install NodeJS))
+	$(if $(shell which standard-version),,\
+		$(error No standard-version found in PATH, install with: \
+			npm install -g standard-version))
```

Comment:
I thought the point of `npx` meant you don't have to install a module to your machine? Is this check necessary?

Reply:
Ah, indeed. This was a copy/paste from something rather old. I'll get rid of it :)